terminus-ui/select/src/select.component.spec.ts

```diff
@@ -1012,6 +1056,26 @@ describe(`TsSelectComponent`, function() {
           expect(instance.autocompleteFormControl.value).toEqual(['Arkansas']);
         });
 
+
+        test(`should set single value 2`, () => {
```

Comment:
I was confused by "should set single value 2" when I read it in isolation. It does make sense within the suite. I tend to err toward more clarity though, so I'd be inclined to call the test "should set single value with a second selection" or something along the lines indicating a reset of sorts.